tests/test_fis.py

```diff
@@ -44,3 +44,49 @@ def test_fis_tag(self):
             'experimentTemplate'
         )
         assert experiment['tags'] == {'Location': 'Messina'}
+
+    def test_fis_mark_for_op(self):
```

Comment:
this test needs to assert that the tag was applied to the resource, which you could do by combining the two tests together